### ISSUE TITLE
fix instructions!

### ISSUE THREAD

**Issue by nonchip** [GDWired/GDPaho#9]
> Should be done in root

none of that ever should, and you didn't even specify *which* permission you didn't understand!

there shouldn't even be any need for a global install since you *hopefully* statically link that.

**Comment by jferdelyi** [comment]
Hi
Or I'd be happy for you to help this project with your super skills by making a pull request or something :)

**Comment by nonchip** [comment]
yeah you know, instead of blaming cmake or paho or me for your inability to document or investigate your own mistake (which, yknow, is obviously just that "install" step being aimed at a systemwide location instead of where it belongs), you could've just admitted that "run it as admin" is a really bad idea to just give as a blanket statement and actually *asked* instead of mocking me for stating the obvious you ignored.

after all you:
> love to learn
